hedera-node/hedera-file-service/src/main/java/com/hedera/node/app/service/file/FileMetadata.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.file;
+
+import com.hedera.hapi.node.base.KeyList;
+import com.hedera.hapi.node.base.Timestamp;
+import com.hedera.pbj.runtime.io.buffer.Bytes;
+
+// TODO : Remove use of TopicMetadata and change to use File instead
+
+/**
+ * File metadata
+ *
+ * @param fileNumber               file's memo
+ * @param expirationTimestamp           file's expiration time
+ * @param keys                     file's sign keys
+ * @param contents                 file bytes that are the contents of the file
+ * @param memo                     file's memo
+ * @param deleted                  file's deleted flag
+ */
+public record FileMetadata(
+        long fileNumber, Timestamp expirationTimestamp, KeyList keys, Bytes contents, String memo, boolean deleted) {}
```

Comment:
can you add null annotations to the params?

Reply:
sure sorry missed it